### DISCUSSION TITLE
Unlink a main account from an organization without logging in

### DISCUSSION THREAD

**Discussion by path01** [community/community#29940] [Other Feature Feedback, Questions, & Ideas]
Hi everyone!

Thanks in advance for reading my issue.  

I have 3 Github accounts.  One of my main accounts is linked to an organization of which I am no longer a part, and the organizational id is "Primary."  When I try to reset the password, it sends the verification to the organizational email.  Any other actions I look up start with "Click on your profile."  

Is there a way to get this id out of a hostage state?

Joy

**Comment by airtower-luna** [comment]
I'm not sure, if anyone can help it [GitHub support](https://support.github.com/request), us other users definitely can't. Note however that having multiple free accounts is against the GitHub ToS. :sweat_smile:
